docs/config/README.md

```diff
@@ -1657,13 +1657,13 @@ To enable it, set `disabled` to `false` in your configuration file.
 
 ### Options
 
-| Option      | Default                | Description                                              |
-| ----------- | ---------------------- | -------------------------------------------------------- |
-| `threshold` | `75`                   | Hide the memory usage unless it exceeds this percentage. |
-| `format`    | `"via $symbol [${ram}( | ${swap})]($style) "`                                     | The format for the module. |
-| `symbol`    | `"🐏"`                 | The symbol used before displaying the memory usage.      |
-| `style`     | `"bold dimmed white"`  | The style for the module.                                |
-| `disabled`  | `true`                 | Disables the `memory_usage` module.                      |
+| Option      | Default                                       | Description                                              |
+| ----------- | --------------------------------------------- | -------------------------------------------------------- |
+| `threshold` | `75`                                          | Hide the memory usage unless it exceeds this percentage. |
+| `format`    | `"via $symbol [${ram}( | ${swap})]($style) "` | The format for the module.                               | $
```

Comment:
Well, the original table was broken but this one is too. It has an extra column. Where did that dollar symbol come from? :confused: